scripts/qa_crowbarsetup.sh

```diff
@@ -2170,27 +2174,34 @@ function oncontroller_testsetup()
     nova list
     glance image-list
 
-    if glance image-list | grep -q SP3-64 ; then
-        glance image-show SP3-64 | tee glance.out
+    image_name="SP3-64"
```

Comment:
please make this variable `local` and i personally would put this into a separate commit, as it does not really relate to the docker enablement